dev-python/scikit-build/scikit-build-0.10.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Improved build system generator for Python C/C++/Fortran/Cython extensions"
+HOMEPAGE="https://github.com/scikit-build/scikit-build"
+SRC_URI="https://github.com/scikit-build/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~amd64 ~x86"
+
+PATCHES="${FILESDIR}/${P}-skip-sandbox-violation-tests.patch"
+
+distutils_enable_sphinx docs dev-python/sphinx_rtd_theme dev-python/sphinx-issues
+distutils_enable_tests pytest
```

Comment:
I am hitting the errors of 
```
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/lib64/python3.6/site-packages/_pytest/config.py", line 379, in _importconftest
    mod = conftestpath.pyimport()
  File "/usr/lib64/python3.6/site-packages/py/_path/local.py", line 668, in pyimport
    __import__(modname)
  File "/var/tmp/portage/dev-python/scikit-build-0.10.0/work/scikit-build-0.10.0/tests/__init__.py", line 22, in <module>
    from mock import patch
ModuleNotFoundError: No module named 'mock'
ERROR: could not load /var/tmp/portage/dev-python/scikit-build-0.10.0/work/scikit-build-0.10.0/tests/conftest.py
```
It seems that >=dev-python/mock-3 is needed.

After installing mock, I have got the following errors:

```
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.6/pytest", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 3241, in <module>
    @_call_aside
  File "/usr/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 3225, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 3254, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 585, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 598, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/lib64/python3.6/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'zipp>=0.5' distribution was not found and is required by importlib-metadata
 * ERROR: dev-python/scikit-build-0.10.0::gentoo failed (test phase):
```

Reply:
Oops, sorry, I forgot to add the test-deps (again) :(

I have added them as upstream specifies them in `requirements-dev.txt`

'zipp>=0.5'  should be pulled in by path-py and pytest